### ISSUE TITLE
Analytics Dashboard Queries

### ISSUE THREAD

**Issue by alexyoung23j** [hicommonwealth/commonwealth#3020]
Queries for real time data into the [analytics dashboard](https://docs.google.com/spreadsheets/d/1fR4aejHVpSBOeLJPSrHnkq-ZA1vLRuGedmDDsQu4nh8/edit?pli=1#gid=1291563958)

**Comment by alexyoung23j** [comment]
Blocked on #2956 getting merged

**Comment by alexyoung23j** [comment]
Duplicate of #3865
